rdmd/eval.d

```diff
@@ -0,0 +1,122 @@
+/**
+Functionality to generate code for the --eval and --loop flags
+*/
+module rdmd.eval;
+
+import std.typecons : Flag;
+
+// For --eval and --loop
+private immutable string importWorld = "
+module temporary;
```

Comment:
a module declaration should not be part of an import list

Reply:
I don't disagree, but once more, that code is not the creation of this PR ;-)

Actually it's probably the least objectionable thing about the `importWorld` string.  Much more of a problem is the great big list of hardcoded import statements, which would break if any module were ever _removed_ from phobos, and to which newer modules cannot be added without breaking compatibility with earlier dmd/phobos releases.

There's probably some straightforward traits-y stuff one could do to fix that and make sure that each module import is conditional on that module actually existing.